README.md

```diff
@@ -10,7 +10,14 @@ This server looks for Bürgeramt appointment every few seconds, and broadcasts t
 pip install -r requirements.txt
 python3 appointments.py
 ```
+### Telegram Notification
 
+Now it is possible to get notified via telegram regarding the appointment dates along with the appointment url. Just click on it and book it right away.
+
+- Go to config directory and rename the config.bak.json to config.json
```

Comment:
```suggestion
- Go to config directory and rename the config.example.json to config.json
```

This might provide more clarity about the file vs bak implying a backup